src/scene/mesh-instance.js

```diff
@@ -370,12 +370,17 @@ class MeshInstance {
         this.mesh.prepareRenderState(renderStyle);
     }
 
+    /**
+     * Gets the render style of the mesh instance.
+     *
+     * @type {number}
+     */
     get renderStyle() {
         return this._renderStyle;
     }
 
     /**
-     * The graphics mesh being instanced.
+     * Sets the graphics mesh being instanced.
```

Comment:
This makes it sounds like mesh instancing is taking place here, I wonder if we could use clearer wording.

Reply:
It's the same wording as before, so in the interest in getting this merged, I'll save that for a follow up PR. 😉